src/style.css

```diff
@@ -1,5 +1,17 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Mochiy+Pop+One&display=swap');
```

Comment:
- [ ] I'm not able to see any styling on your project as you can see in the image below, Please fix this, As you can see in the image below.
    - Kindly check your deployed project, it has the same issue 🚀